spec/fixtures/medium_short_fixture_ids.csv

```diff
@@ -0,0 +1,28 @@
+oid
```

Comment:
medium short? feels like there has to be a more purpose focused name for this file

Reply:
Agree, and should be in a different PR as well